File: ProgrammingFundamentals/C#/03.DataTypesAndVariables/Exercises/P12_RectangleProperties/P12_RectangleProperties.cs
using System;

namespace P12_RectangleProperties
{
    class P12_RectangleProperties
    {
        static void Main(string[] args)
        {
            double width = double.Parse(Console.ReadLine());
            double height = double.Parse(Console.ReadLine());

            // Calculating - perimeter - area - diagonal
            Console.WriteLine((2 * width) + (2 * height) + Environment.NewLine + (width * height) + Environment.NewLine +
                                Math.Sqrt((width * width) + (height * height)));
        }
    }
}

File: ProgrammingFundamentals/Java/03.DataTypesAndVariables/Exercises/src/P12_RectangleProperties.java
import java.text.DecimalFormat;
import java.util.Scanner;
public class P12_RectangleProperties {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);

        double width = Double.parseDouble(console.nextLine());
        double height = Double.parseDouble(console.nextLine());
        DecimalFormat df = new DecimalFormat("#.#############");

        // Calculating - perimeter - area - diagonal
        System.out.println(df.format(((2 * width) + (2 * height))) + System.lineSeparator() +
                df.format((width * height)) + System.lineSeparator() +
                df.format(Math.sqrt((width * width) + (height * height))));
    }
}
